### PR TITLE
Adding Crypto.org Chain v4 upgrade instructions

### PR DESCRIPTION
**Pull request by lezzokafka**
To get ready for V4 upgrade, in this PP, we provide:
- Crypto.org Chain v4 upgrade instructions;
- Suggested config setting.

### CODE REVIEW

File: docs/getting-started/upgrade_guide_v4.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+# The default value is 10MB.
+max-recv-msg-size = "10485760"
+
+# MaxSendMsgSize defines the max message size in bytes the server can send.
+# The default value is math.MaxInt32.
+max-send-msg-size = "2147483647"
+```
+
+::: tip Remarks:
+If you are upgrading from `v3.3.4` or earlier, kindly also add  the following fields to the `app.toml` under `Base Configuration`:
```
Comment (zkaizhi) [inline docs/getting-started/upgrade_guide_v4.md]: Might also link to https://crypto.org/docs/getting-started/mainnet.html#step-0-notes-on-network-upgrades
as it contains explanation about the iavl setting?

**Review by zkaizhi — COMMENTED** [review]
Do we want to add the upgrade guides btw also in the sidebar? /.vuepress/config.js to make it easier to find these pages?

**Review by zkaizhi — APPROVED** [review]

**Comment by lezzokafka** [comment]
Closing this and replaced by a verified version.

**Pull request closed without merging**